lib/shortenator.rb

```diff
@@ -70,12 +71,12 @@ def valid_link?(link)
       response.code.to_i == 200
     end
 
-    def shorten_link(link, client)
+    def shorten_link(link, client, group_guid: nil)
       retries = 0
       link = replace_localhost(link) if link.include? 'localhost'
       loop do
         begin
-          bitly_response = client.create_bitlink(long_url: link, tags: @all_tags)
+          bitly_response = client.create_bitlink(long_url: link, tags: @all_tags, group_guid: group_guid)
```

Comment:
Does the behavior and result within Bitly change here between currently not passing `group_guid` at all to passing it as `nil`?

Reply:
I have more on this in the [bitly doc](https://leafly.atlassian.net/wiki/spaces/RT/pages/979271887/Bitly+Documentation#Groups-In-Depth), but if it's nil, it will use the default group on the account.